tests/foreman/api/test_contentmanagement.py

```diff
@@ -949,19 +947,18 @@ def test_positive_mirror_on_sync(self, capsule_vm):
         )[0]
         activation_key.add_subscriptions(data={'subscription_id': subscription.id})
         # Subscribe a host with activation key
-        with VirtualMachine(distro=DISTRO_RHEL7) as client:
-            client.install_katello_ca()
-            client.register_contenthost(org.label, activation_key.name)
-            # Install the package
-            package_name = FAKE_1_YUM_REPO_RPMS[2].rstrip('.rpm')
-            result = client.run(f'yum install -y {package_name}')
-            assert result.return_code == 0
-
-            # Ensure package installed
-            result = client.run(f'rpm -qa | grep {package_name}')
-
-            assert result.return_code == 0
-            assert package_name in result.stdout[0]
+        rhel7_contenthost.install_katello_ca()
+        rhel7_contenthost.register_contenthost(org.label, activation_key.name)
+        # Install the package
+        package_name = FAKE_1_YUM_REPO_RPMS[2].rstrip('.rpm')
+        result = rhel7_contenthost.run(f'yum install -y {package_name}')
+        assert result.return_code == 0
```

Comment:
I guess it should be like
```suggestion
        assert result.status == 0
```